backend/lib/plugins/auth.js

```diff
@@ -53,6 +86,16 @@ const authPlugin = async (app) => {
     }
   });
 
+  /* eslint-disable func-names */
```

Comment:
why did you need the ESLint disable? Seems like we already have a lot of anonymous callback functions in other places

Reply:
Working on a few front-end improvements now. `func-names` only errors for non-arrow functions as far as I can tell. I need non-arrow functions here for `this`.